api/all/src/main/java/io/opentelemetry/api/trace/SpanContext.java

```diff
@@ -135,7 +135,9 @@ default boolean isSampled() {
    * @return {@code true} if this {@code SpanContext} is valid.
    */
   default boolean isValid() {
-    return TraceId.isValid(getTraceId()) && SpanId.isValid(getSpanId());
+    return TraceId.isValid(getTraceId())
+        && SpanId.isValid(getSpanId())
+        && getTraceFlags().isValid();
```

Comment:
this is a change from the specification, I believe, which says that the SpanContext is valid iff the spanid and traceid are valid, but doesn't mention trace flags.